table/table_test.cc

```diff
@@ -969,7 +969,7 @@ class HarnessTest : public testing::Test {
     assert(part <= total);
     size_t start_i = (part - 1) * args.size() / total;
     size_t end_i = part * args.size() / total;
-    for (unsigned int i = start_i; i < end_i; i++) {
+    for (unsigned int i = static_cast<int>(start_i); i < end_i; i++) {
```

Comment:
why not static_cast<unsigned int>?

Reply:
Not sure what you mean, I did use static_cast?

Reply:
Thanks for fixing this. Can we cast to "unsigned int" instead?